src/libs/sfdk/sdk.cpp

```diff
@@ -40,6 +40,8 @@
 #include <QSettings>
 #include <QStandardPaths>
 
+#include <libs/sfdk/sfdk_version_p.h>
```

Comment:
This should be just `"sfdk_version_p.h"`

Reply:
Add `INCLUDEPATH` for this